cmd/conformance/aws/main.go

```diff
@@ -81,7 +82,7 @@ func main() {
 			return
 		}
 
-		idx, err := storage.Add(r.Context(), tessera.NewEntry(b))()
+		idx, err := dedupe.Add(r.Context(), tessera.NewEntry(b))()
```

Comment:
I find it a bit weird when calling the `dedupe.Add`. It sounds like it's adding the entry to the dedupe queue instead of the storage. Can we rename `dedupe` to something else like `dedupedStorage`?

Reply:
Done.